types/express/index.d.ts

```diff
@@ -0,0 +1,7 @@
+import { User } from '../../src/user/schemas/user.schema'
+
+declare module 'express-serve-static-core' {
+  interface Request {
+    user: Omit<User, 'password'>
```

Comment:
We're using this type in multiple places (`Omit<User, 'password'>`), let's declare it once somewhere as an alias, then reuse.

Reply:
Fixed